upgrade/scripts/upgrade/prerequisites.sh

```diff
@@ -1339,41 +1328,11 @@ state_recorded=$(is_state_recorded "${state_name}" "$(hostname)")
 if [[ ${state_recorded} == "0" ]]; then
   echo "====> ${state_name} ..." | tee -a "${LOG_FILE}"
   {
-    # make sure cray-nls-postgres is healthy
-    i=1
-    max_checks=40
-    wait_time=15
-    healthy="false"
-    while [[ ${i} -le ${max_checks} ]]; do
-      if [[ "$(kubectl get postgresql cray-nls-postgres -n argo -ojsonpath='{.status.PostgresClusterStatus}')" == "Running" ]]; then
-        echo "cray-nls-postgres is healthy"
-        healthy="true"
-        break
-      fi
-      echo "Waiting for postgres operator to consider cray-nls-postgres healthy (${i}/${max_checks})"
-      sleep ${wait_time}
-      i=$((i + 1))
-    done
-
-    if [[ ${healthy} == "false" ]]; then
-      echo "ERROR: cray-nls-postgres is not healthy after $((max_checks * wait_time)) seconds"
-      exit 1
-    fi
-
-    # CASMINST-6700 retry cray-nls upgrade
```

Comment:
No need to retry cray-nls upgrade. The issue was happening, because posgresql CRDs were not updated during operator upgrade. Now we restart operator and wait for CRD update in FIX_POSTGRES.